packages/chain-adapters/src/evm/EvmBaseAdapter.ts

```diff
@@ -44,6 +44,14 @@ import { getErc20Data } from './utils'
 
 export type EvmChainIds = KnownChainIds.EthereumMainnet | KnownChainIds.AvalancheMainnet
 
+export const isEvmChainId = (
+  maybeEvmChainId: string | EvmChainIds
```

Comment:
I was going to comment on this, but saw `Note, in a future PR I'd like to rename the new EvmChainIds type to the singular, EvmChainId.` 🧠

Reply:
You know what, I'm just going to do it now so it doesn't have to be a breaking change after I introduce it in web: https://github.com/shapeshift/lib/pull/902/commits/bdfe4be9a7aeb4c39ca2fd603e37470483d09466

Reply:
ha, great minds. i made this same naming update on my evm types pr and agree